mlrun/common/schemas/model_endpoints.py

```diff
@@ -226,7 +227,9 @@ def flat_dict(self):
                 # If the value is not from type str or bool (e.g. dict), convert it into a JSON string
                 # for matching the database required format
                 if not isinstance(
-                    model_endpoint_dictionary[k_object][key], (str, bool)
+                    model_endpoint_dictionary[k_object][key], (str, bool, int)
+                ) or isinstance(
+                    model_endpoint_dictionary[k_object][key], (enum.IntEnum)
```

Comment:
```suggestion
                    model_endpoint_dictionary[k_object][key], enum.IntEnum
```
Also, consider extracting `model_endpoint_dictionary[k_object][key]`.

Reply:
Right, updated.